Source/LinqToDB/Internal/DataProvider/ClickHouse/ClickHouseProviderAdapter.cs

```diff
@@ -348,7 +348,9 @@ internal sealed class ClickHouseConnection
 			[Wrapper]
 			public sealed class ClickHouseConnectionStringBuilder : TypeWrapper
 			{
+#pragma warning disable IDE0051 // Remove unused private members
```

Comment:
would generally prefer these to be attributes instead of pragmas...

```suggestion
[SuppressMessage(...)]
```